R/pkg/R/SQLContext.R

```diff
@@ -62,21 +62,23 @@ infer_type <- function(x) {
   }
 }
 
-#' Create a DataFrame from an RDD
+#' Create a DataFrame
 #'
-#' Converts an RDD to a DataFrame by infer the types.
+#' Converts R data.frame or list into DataFrame.
```

Comment:
I'm not sure we should have `RDD` here since `RDD` is no longer exposed in SparkR?